CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.0
```

Comment:
I assume we're calling this a Y because Jetty is the main dependency in this library?

Reply:
Yeah I think so.

Reply:
Yeah, and Jetty doesn't really follow semver. I think this accurately reflects the risk involved with this update.